analyzers/src/ITs.JsonParser/Json/Location.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace ITs.JsonParser.Json;
+
+internal class Location
```

Comment:
This PR aims to map it exactly as it was done by the powershell script, so Location is reused.
I prefer to do the rework you suggest on the next PR, where it will be actually needed.
I will leave a TODO so that it's clear.

Reply:
This is practically just a rename and move. So it would still respect the functionality from the PS.

The ToDo is missing the rename part. I'd do this directly here. Unless I'm missing some functional impact. I expect the `RuleIssues` class to use `SarifLocationFile`. At least it will be clear where it comes from and why we need to change it later.

Reply:
Fair point, I moved it inside `SarifReport`. and re-used it in `RuleIssues`.
Now we only have two classes that jsons are mapped _from_ and _to_, which is pretty handy to "map" the JSON structure visually.

I left a TODO for the next PR to stop re-using the `SarifLocationFile` type to be explicit.